README.md

```diff
@@ -533,6 +571,63 @@ project.getMetadata(function(err, metadata) {
 ```
 
 
+## Google Cloud Speech (Limited Preview)
+
+> **This is a Limited Preview release of Google Cloud Speech.** This feature is not covered by any SLA or deprecation policy and may be subject to backward-incompatible changes.
+
+- [API Documentation][gcloud-speech-docs]
+- [Official Documentation][cloud-speech-docs]
+
+```js
+var gcloud = require('gcloud');
+
+// Authorizing on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authorization section above).
+
+var speech = gcloud.speech({
+  projectId: 'my-project',
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// Detect the speech in an audio file.
+speech.detect('./audio.raw', {
+  encoding: 'LINEAR16',
+  sampleRate: 16000
+}, function(err, result) {
+  // result = {
+  //   transcript: 'how old is the Brooklyn Bridge',
+  //   confidence: 0.97234234234
+  // ]
+});
+```
+
+```js
+// Detect the speech in an audio file stream.
+var initialRequest = {
+  encoding: 'LINEAR16',
+  sampleRate: 16000,
+  continuous: true
+};
+
+var stream = speech.recognize(initialRequest);
+  .on('data', function (recognizeResponse) {
+    console.log(recognizeResponse);
+  });
+
+stream.write({
+  audioRequest: {
```

Comment:
I don't think the Job naming convention makes sense. Let's pick other names.

I don't think using Sync/Async in a way that doesn't line up with the same meaning in Node makes sense either. Let's pick other names.

> In the v1beta1 API, the methods are named SyncRecognize, AsyncRecognize, and StreamingRecognize, and those names will almost certainly carry forward to GA. Could we follow those same naming conventions in gcloud?

I think we're crossing wires here. Our job isn't to thin wrap the Speech API and have it line up 1:1 with your protos. If you want that, there will be auto-generated code. 

The way I see it, there's one verb of stuff to do here: recognize some audio. Is that right?

If so, then there are two situations we could be in:

1. I have all the data that I want to recognize right here in a file.
2. I don't have all the data that I want to recognize because it's a live stream coming in.

For the first:

```js
speech.recognize({ info about the data, including a URI for where to get it or the raw bytes })
  .then(doSomethingWithRecognizedData);
  // or .on('data', ...); but you get the point.
```

```js
speech.recognizeFromStream({ info about the data, no URI / bytes })
speech.pipe(stream)
speech.on('data', doSomethingWithDataRecognizedSoFar);
```

I don't want to think about InitialRequest objects, RecognizeRequest objects, or any of the RPCs. I want to either give you bytes or a stream of bytes I'm figuring out, and you give me back either the recognized stuff in one go, or the latest value of stuff that you've recognized so far...

Am I crazy?